app/views/reference/index.njk

```diff
@@ -15,29 +15,20 @@
   <form action="/pay/reference/{{ productExternalId }}" method="post" name="referencePaymentForm" class="push-bottom">
     <input id="csrf" name="csrfToken" type="hidden" value="{{ csrf }}"/>
     <div class="form-group">
-      <label id="product-reference-label" for="payment-reference" class="form-label-bold">
-        <span id="product-reference-label-span">{{ productReferenceLabel }}</span>
-        <span id="product-reference-label-hint" class="form-hint">{{ productReferenceHint }}</span>
+      <label id="payment-reference-label" class="form-label form-label-bold" for="payment-reference">
+        {{ paymentReferenceLabel }}
+        {% if paymentReferenceHint %}
+          <span id="payment-reference-hint" class="form-hint">{{ paymentReferenceHint }}</span>
+        {% endif %}
       </label>
-      {% if productReference %}
-        <div class="panel panel-border-wide">
-          <h2 id="payment-reference" class="lede">{{ paymentReference }}</h2>
-        </div>
```

Comment:
I assume removing this has also been agreed

Reply:
This is a comment from @SandorArpa where a `reference` will either exist and will not reach this page or will be missing and needs to display this page. So this check is unnecessary :)